frontend/tests/acceptance/features/webUIAccount/updatePassword.feature

```diff
@@ -0,0 +1,46 @@
+Feature: update the account password
+  As a user
+  I want to update my account password
+  So that I can log into my account using new password
+
+  Background:
+    Given the user has signed up with name "user1", email "user1@gmail.com" password "password"
+    And the user has logged in with email "user1@gmail.com" and password "password"
+    And the user has browsed to the account page
+
+  @blankPasswordUpdate
+  Scenario Outline: update password with blank credentials
+    When the user tries to update password with the following credentials using the webUI
+      | currentPassword   | newPassword   | confirmPassword   |
+      | <currentPassword> | <newPassword> | <confirmPassword> |
+    Then the user should stay on the account page
+    And the user must be able to login with email "user1@gmail.com" and password "password"
```

Comment:
checking for empty fields doesn't changes anything. So, this line may not be necessary.